cross/i2pd/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = i2pd
+PKG_VERS = 2.18.0
+PKG_EXT = tar.gz
+PKG_DIST_SITE = https://github.com/PurpleI2P/i2pd/archive
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
```

Comment:
Please use `PKG_DIST_NAME = $(PKG_DIR).$(PKG_EXT)` after `PKG_DIR` is set, so that downloaded archive and digests does not conflict with any other tool